src/Evolution/Systems/Burgers/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(LIBRARY Burgers)
 set(LIBRARY_SOURCES
   Equations.cpp
   Fluxes.cpp
+  Minmod.cpp
```

Comment:
Do you think we should generalize this to `LimiterInstantiations.cpp` and instantiate all limiters there?

Reply:
As I see it: the advantages to grouping the instantiations into a single file are having fewer files and making it easier to see what limiters are available for one system; the downside is having to re-instantiate all limiters when one limiter changes. Are there other considerations that I am missing?

At the moment I do not have a strong opinion on the matter, so I am happy to make the change if you prefer it. Just let me know what you think :slightly_smiling_face:

Reply:
Those are the only two things I can think of. Given that the number of limiters we will have will be < 10 (or at least given the current literature it seems unlikely to exceed 10) I think having separate files so that the compilation can be parallelized better is a good thing :)